src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -574,6 +588,34 @@ private void validateSequenceDictionaries() {
             }
         }
 
+        // Check the master dictionary against the reference / reads / features
```

Comment:
I think we should check the master dictionary against the other dictionaries before the existing checks, rather than after them.